scripts/scil_merge_json.py

```diff
@@ -14,25 +14,30 @@
                              assert_inputs_exist, assert_outputs_exist)
 
 
-def _merge_dict(dict_1, dict_2):
+def _merge_dict(dict_1, dict_2, no_list=False, recursive=False):
```

Comment:
I'm lost in your merge :)

Reply:
Yeah, this is gigantic clusterf**k to replace the even bigger clusterf**k of jq in tractometry flow.

I think the best way is to try it ;)
But it is oriented at 99% tractometry-flow and nothing else